app/views/doctors/transcripts/index.haml

```diff
@@ -1,5 +1,27 @@
 - if @transcripts.present?
-  - @transcripts.each do |transcript|
-    = transcript.chat_room_id
+  %h3 Your Previous Transcripts
+  %table.table.table-hover
+    %thead
+      %tr
+        %th Patient
+        %th Description
+        %th Transcript creation time 
+        %th 
+
+    %tbody
+      - @transcripts.each do |transcript|
+        - if transcript.chat_room.urgent_request.description.present?
```

Comment:
If you have a look at `app/models/chat_room.rb:12`, you'll see the method called request. You can use this function to completely remove this if statement.

Instead of
```
if urgent request
  do urgent request stuff
else
  do consultation request stuff
```

You can just do:
```
%td= transcript.chat_room.patient.name           
%td= transcript.chat_room.request.description         
%td= transcript.updated_at
%td= link_to 'Show', patient_transcript_path(id: transcript)
```